### PR TITLE
Leona/neon install 2

### PR DESCRIPTION
**Pull request by indie**

### CODE REVIEW

File: doc/sphinx/source/install.rst [hunk 1]
```diff
@@ -93,6 +95,55 @@ The process documented here will work on Ubuntu\* 16.04 (LTS)
...
+
+      # mkdir -p /opt/libraries
+      # chown -R username:username /opt/libraries
+
+#. Update the system with :command:`yum` and issue the following commands: 
+   
+   .. code-block:: console
+
+      # yum update
+      # yum -y install zlib-devel
```
Comment (aprocter) [inline doc/sphinx/source/install.rst]: Any reason this needs to be done in a separate command from the one that follows? (just idly curious)

File: doc/sphinx/source/install.rst [hunk 2]
```diff
@@ -93,6 +95,55 @@ The process documented here will work on Ubuntu\* 16.04 (LTS)
...
+      # yum install ncurses-libs ncurses-devel patch diffutils wget 
+      gcc-c++ make git perl-Data-Dumper
+
+
+#. Install Cmake 3.4:
```
Comment (aprocter) [inline doc/sphinx/source/install.rst]: FWIW it is possible to run `cmake` out of the build directory, which eliminates the need to `su`. Will leave it up to you whether it's worth documenting that here. (We might just mention that it's possible but say it's beyond the scope of this document or something...)

File: doc/sphinx/source/install.rst [hunk 3]
```diff
@@ -93,6 +95,55 @@ The process documented here will work on Ubuntu\* 16.04 (LTS)
...
+#. As non-root users, you can now clone the `NervanaSystems` ``ngraph`` repo 
+   and use Cmake 3.4.3 to install the nGraph libraries to ``$HOME/ngraph_dist``.
+
+   .. code-block:: console
+
+      $ cd /opt/libraries 
+      $ git clone git@github.com:NervanaSystems/ngraph.git
+      $ mkdir build && cd build
+      $ cmake ../
+      $ make
```
Comment (aprocter) [inline doc/sphinx/source/install.rst]: `make install` is missing

File: doc/sphinx/source/install.rst (hunk 1 above)
Comment (indie) [reply doc/sphinx/source/install.rst]: Just the whole long list of commands (3 + lines of wrapping) seemed like a bit much. Visually it's nicer to not have to use horizontal scroll to to see or grab commands

File: doc/sphinx/source/install.rst [hunk 4]
```diff
@@ -93,6 +95,53 @@ The process documented here will work on Ubuntu\* 16.04 (LTS)
...
+      # cd cmake-3.4.3
+      # ./bootstrap
+      # make && make install  
+
+#. As non-root users, you can now clone the `NervanaSystems` ``ngraph`` repo 
+   and use Cmake 3.4.3 to install the nGraph libraries to ``$HOME/ngraph_dist``.
+
+   .. code-block:: console
+
+      $ cd /opt/libraries
```
[the remark this answers is not in the document]
Comment (indie) [reply doc/sphinx/source/install.rst]: The `opt/libraries` directory suggestion needs to stay for consistency purposes in the documentation.  A lot of the documentation examples work via the clone, and I need a way to produce reproducible results.  Being explicit here is the only way to create a path that I am sure a user can follow to get the same results.

File: doc/sphinx/source/install.rst [hunk 5]
```diff
@@ -93,6 +95,53 @@ The process documented here will work on Ubuntu\* 16.04 (LTS)
...
+      # make && make install  
+
+#. As non-root users, you can now clone the `NervanaSystems` ``ngraph`` repo 
+   and use Cmake 3.4.3 to install the nGraph libraries to ``$HOME/ngraph_dist``.
+
+   .. code-block:: console
+
+      $ cd /opt/libraries 
+      $ git clone git@github.com:NervanaSystems/ngraph.git
+      $ mkdir build && cd build
```
[the remark this answers is not in the document]
Comment (indie) [reply doc/sphinx/source/install.rst]: Good catch, thanks.

File: doc/sphinx/source/install.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (postrational) [reply doc/sphinx/source/install.rst]: ngraph uses `$HOME` for this purpose. You could make paths relative to `$HOME` or simply `~`

File: doc/sphinx/source/install.rst [hunk 6]
```diff
@@ -93,6 +95,53 @@ The process documented here will work on Ubuntu\* 16.04 (LTS)
...
+      # ./bootstrap
+      # make && make install  
+
+#. As non-root users, you can now clone the `NervanaSystems` ``ngraph`` repo 
+   and use Cmake 3.4.3 to install the nGraph libraries to ``$HOME/ngraph_dist``.
+
+   .. code-block:: console
+
+      $ cd /opt/libraries 
+      $ git clone git@github.com:NervanaSystems/ngraph.git
```
[the remark this answers is not in the document]
Comment (indie) [reply doc/sphinx/source/install.rst]: Hmm..... SSH *does* work in Intel; as I mentioned to you before, if you have your system configured correctly (I emailed you a tip for how to set it up).  To me it seems distracting to branch out to explain the differences between SSH vs HTTPS in cloning a repo at this point in what we are doing here.  Again, I will leave the SSH instructions and add a note that people can also use HTTPS to clone, if they have trouble with SSH/

[the remark this answers is not in the document]
Comment (postrational) [reply doc/sphinx/source/install.rst]: I don't think you need to explain anything. `git clone git` and `git clone https` are both self-explanatory.

File: doc/sphinx/source/install.rst [hunk 7]
```diff
@@ -93,6 +95,53 @@ The process documented here will work on Ubuntu\* 16.04 (LTS)
...
+
+      # mkdir -p /opt/libraries
+      # chown -R username:username /opt/libraries
+
+#. Update the system with :command:`yum` and issue the following commands: 
+   
+   .. code-block:: console
+
+      # yum update
+      # yum -y install zlib-devel install ncurses-libs ncurses-devel patch diffutils wget gcc-c++ make git perl-Data-Dumper
```
[the remark this answers is not in the document]
Comment (indie) [reply doc/sphinx/source/install.rst]: Agree, fixed.

**Pull request merged**